make/autoconf/flags-cflags.m4

```diff
@@ -586,11 +586,11 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
 
   # CXXFLAGS C++ language level for all of JDK, including Hotspot.
   if test "x$TOOLCHAIN_TYPE" = xgcc || test "x$TOOLCHAIN_TYPE" = xclang; then
-    LANGSTD_CXXFLAGS="-std=c++14"
```

Comment:
Is this really an intention on your part, to bump the C++ level inside a PR to remove unused code? 😕 Or is it a merge error from a different PR?

Reply:
It's a checkout error from the original gigantic Pull Request, sorry. I don't plan to raise the JDK to C++17 in this one. But I'm not sure what to do with this now, since maybe\_unused is C++17. I don't think anyone is going to appreciate [[gnu::unused]] inside Windows code

Reply:
Then you'll have to create a macro for disabling the unused warnings (however, I'm not sure how well that would be received). Or fix the bug if it is actually unused. Or wait until we go to 17.

Reply:
I don't think it's a bug since the error is an unused pointer typedef, and it seems customary to provide the pointer typedef alongside the actual struct in awt code, even if not needed. I could just disable the warning for the file in the port, but that way no more potential bugfixes in that file would be possible

Reply:
@magicus Sorry for the ping, but I'm a bit confused. Do you know what bug report site Phil is referring to when he says to submit a bug to AWT instead? Is it just the JBS? (There's a little bit of a problem if it is, it is extremely inconvenient to review what to do with the problem areas in the JBS as opposed to a Pull Request, that and no one actually checks the JBS for new issues, so a bug submitted there is doomed to be ignored until the end of time)

Reply:
You need to split this up into multiple parts. One part is about removing dead code. Do not comment it out, just remove it. Open a new JBS issue on client-libs for removal of dead code. This should be trivial to get pushed.

Then, you have some other changes. Looks like you moved something from a header file to a cpp file. Make that a separate PR, also on client-libs.

Then you are making some annotation stuff. I have not seen these before. Are they established? This might need some more discussion to get a consensus on how to proceed with.

And finally you are bumping the C++ language level. You have already opened a separate JBS issue for that, and we've said that we do that when we do that. So just drop that part.

Reply:
The dead code removal may not be correct, same goes for the rest of the changes. Most, if not all of them (Especially the C\+\+17 change) are pure placeholders to simply draw attention to the problem sites. I think all of them need discussion on how to proceed with (Same with the corresponding jdk.accessibility changes), not just the annotations. Speaking of which, the annotations are C\+\+ attributes, introduced in C\+\+11, and are (fortunately) Standard and not compiler extensions. Splitting this up to even smaller changes may or may not inconvenience the few reviewers that AWT and A11Y has, which is what I'm worried about

Reply:
The code has likely been dead for years. If no-one has reported any bugs about this, it is likely correct that it is unused.

If you think that is a bug, do some git archaeology to determine when it became dead, or locate what pathway you think it should be used it, and try to provoke an error. 

I still think publishing a simple PR which just removes dead code is the best way for you to proceed here. Such a PR should be trivial to review.